src/WeekThree/Homework3/FizzBuzz.java

```diff
@@ -17,16 +17,18 @@ public static void main(String[] args){
     public static void fizzBuzz(int num){
 
         for(int i = 1; i <= num; i++){
-//            System.out.println(i);
           if(i % 3 == 0 && i %5 == 0){
                 System.out.println("Number: " + i + " is Fizz Buzz!");
             }
-            else if(i %3 == 0){
+            else if(i %3 == 0 ){
                 System.out.println("Number: " + i + " is Fizz!");
             }
-            else if(i %5 == 0){
+            else if(i %15 == 0){
```

Comment:
@KurisuLim you probably meant 5 not 15, right?

Reply:
@azakuanov 
Sorry I was meant to change the other if. It is fixed now.